doc/sphinx/reference/varnishncsa.rst

```diff
@@ -50,19 +74,23 @@ Supported formatters are:
 
 %b
   Size of response in bytes, excluding HTTP headers.  In CLF format,
-  i.e. a '-' rather than a 0 when no bytes are sent.
+  i.e. a '-' rather than a 0 when no bytes are sent.  For backend mode,
```

Comment:
I'd like these descriptions to be less like backend mode is just something bolted on. Maybe along the lines of:
"In client mode, the number of bytes excluding HTTP headers sent to the client. In backend mode the number of bytes excluding HTTP headers received from the backend. In CLF format..."